From e222fb728217ca60790ef5792815edb1d436922d Mon Sep 17 00:00:00 2001
From: Steve Loughran <stevel@apache.org>
Date: Tue, 15 May 2018 16:19:03 +0100
Subject: [PATCH 2805/2863] HADOOP-15466. Correct units in adl.http.timeout.
 Contributed by Sean Mackrory,

(cherry picked from commit 07d8505f75ec401e5847fe158dad765ce5175fab)

Change-Id: Ibd43998c1babd7d84aa3dff0e83b1d0112aaf1aa
---
 .../src/main/resources/core-default.xml            |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml b/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
index 9deacb6..9f2aa11 100644
--- a/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
+++ b/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
@@ -2452,8 +2452,8 @@ for ldap providers in the same way as above does.
     <name>adl.http.timeout</name>
     <value>-1</value>
     <description>
-      Base timeout (in seconds) for HTTP requests from the ADL SDK. Values of
-      zero or less cause the SDK default to be used instead.
+      Base timeout (in milliseconds) for HTTP requests from the ADL SDK. Values
+      of zero or less cause the SDK default to be used instead.
     </description>
   </property>
 
-- 
1.7.9.5

